iceoryx2-bb/posix/tests/udp_socket_tests.rs

```diff
@@ -71,76 +68,82 @@ fn udp_socket_send_receive_works() {
 
 #[test]
 fn udp_socket_server_with_same_address_and_port_fails() {
-    let _sut_server_1 = UdpServerBuilder::new()
+    let sut_server_1 = UdpServerBuilder::new()
         .address(Ipv4Address::new(127, 0, 0, 1))
-        .port(Port::new(55555))
         .listen()
         .unwrap();
 
     let sut_server_2 = UdpServerBuilder::new()
         .address(Ipv4Address::new(127, 0, 0, 1))
-        .port(Port::new(55555))
+        .port(sut_server_1.port())
         .listen();
 
     assert_that!(sut_server_2.err().unwrap(), eq UdpServerCreateError::AddressAlreadyInUse);
 }
 
 #[test]
 fn udp_socket_when_socket_goes_out_of_scope_address_is_free_again() {
+    let port;
     {
-        let _sut_server_1 = UdpServerBuilder::new()
+        let sut_server_1 = UdpServerBuilder::new()
             .address(ipv4_address::LOCALHOST)
-            .port(Port::new(55555))
             .listen()
             .unwrap();
+        port = sut_server_1.port();
     }
 
     let sut_server_2 = UdpServerBuilder::new()
         .address(ipv4_address::LOCALHOST)
-        .port(Port::new(55555))
+        .port(port)
         .listen();
 
     assert_that!(sut_server_2, is_ok);
 }
 
 #[test]
 fn udp_socket_server_has_correct_address() {
+    let port = {
+        UdpServerBuilder::new()
+            .address(ipv4_address::LOCALHOST)
+            .listen()
+            .unwrap()
+            .port()
+    };
```

Comment:
I guess you never imagined to hear this words from me, but the braces can be omitted ;)

Reply:
Finally, the day has come.